membership-attribute-service/app/services/ContributionsStoreDatabaseService.scala

```diff
@@ -1,60 +1,63 @@
 package services
 
 import anorm._
-import cats.data.EitherT
-import cats.syntax.applicativeError._
-import cats.instances.future._
 import com.typesafe.scalalogging.StrictLogging
 import models.ContributionData
 import play.api.db.Database
+import services.DatabaseService.DatabaseGetResult
 
 import scala.concurrent.{ExecutionContext, Future}
+import scala.util.control.NonFatal
+import scalaz.{EitherT, \/}
```

Comment:
reluctantly switching to scalaz here because the calling code uses it heavily, and it would get really messy otherwise.
It would be nice to rip out scalaz completely, there's no reason to use `\/` these days